sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-disbursements-report-bc-grants.sql

```diff
@@ -0,0 +1,72 @@
+UPDATE
+    sims.report_configs
+SET
+    report_sql = (
+        '(select
```

Comment:
Please format the query in same way like it's original form. 

![image](https://github.com/user-attachments/assets/512a3da7-6db1-4438-b7dc-955e21c94cc2)